jetty/src/io/pedestal/http/jetty.clj

```diff
@@ -94,15 +98,24 @@
       ;; In 0.7 and earlier, was a namespaced key
       (::http-configuration options)
       ;; Otherwise, build on the fly
-      (let [{:keys [insecure-ssl?]} options
-            http-conf ^HttpConfiguration (HttpConfiguration.)]
+      (let [{:keys [sni-host-check? insecure-ssl?]} options
+            http-conf ^HttpConfiguration (HttpConfiguration.)
+            sni-enabled? (cond
+                           (and (nil? sni-host-check?)
+                                (nil? insecure-ssl?))   true
+                           (not (nil? sni-host-check?)) sni-host-check?
+                           :else                        (not insecure-ssl?))]
+        (when insecure-ssl?
+          (deprecated :insecure-ssl? :in "0.8.0" :noun "prefer :sni-host-check? which is more specific but inverse to :insecure-ssl? which"))
```

Comment:
I think the :noun should be `:insecure-ssl? Jetty option`. Update the docstring for the `server` function to discuss the upgrade path.